library/Garp/Model/Behavior/HtmlFilterable.php

```diff
@@ -44,23 +50,35 @@ public function filter($string, HTMLPurifier_Config $config = null) {
         if (!$string) {
             return $string;
         }
-
-        if (!$config) {
-            $config = $this->getConfig();
-        }
-
+        $config->finalize();
```

Comment:
Als we dit weghalen (dat `if` statement) is het wel een goed idee om die parameter niet langer optioneel te maken.
Anders zou je `finalize()` kunnen aanroepen op `null`.